gen/field.go

```diff
@@ -244,6 +248,37 @@ func (f *fieldGroupGenerator) declFieldName(fs *compile.FieldSpec) (string, erro
 	return name, nil
 }
 
+func (f fieldGroupGenerator) DefineDefaultConstructor(g Generator) error {
+	var hasDefaults bool
+	for _, f := range f.Fields {
+		if f.Default != nil {
+			hasDefaults = true
+			break
+		}
+	}
+	if !hasDefaults {
+		return nil
+	}
+	return g.DeclareFromTemplate(
+		`
+		// Default_<.Name> constructs a new <.Name> struct,
+		// pre-populating any fields with defined default values. 
+		func Default_<.Name>() *<.Name> {
+			<- $v := newVar "v" ->
+			v := &<.Name>{}
+
+			<range .Fields>
+				<- $fname := goName . ->
+				<- if .Default ->
+					<$v>.<$fname> = <constantValuePtr .Default .Type>
+				<end>
+			<end>
+
+			return v
```

Comment:
with var change above, this becomes a pointer:

```suggestion
			return &v
```